### ISSUE TITLE
The bot referenced by the \'from\' field is unrecognized

### ISSUE THREAD

**Issue by vianasw** [Microsoft/BotBuilder#2450]
Hi, I'm having an issue trying to send messages through Direct Line from my bot in node.js, with the botbuilder version 3.7. The error I'm getting is 400 Bad Request, which is not very explanatory, but after inspecting the response object in the ChatConnector class I've found the actual error:
```
{ error:
      { code: 'MissingProperty',
        message: 'The bot referenced by the \'from\' field is unrecognized' } } }
```

And inspecting the request object I've noticed that the from seems to be correct:

` body: '{"type":"message","text":"Some bot message","from":{"id":"my_bot_handle","name":"My Bot Name"},"recipient":{"id":"DirectLineClient"}}'
`

I've compared the exact same request to another bot that I have with the same account but with different app id and password and I saw no difference whatsoever, and I'm using Direct Line version 1.1 for both bots.

Am I missing something?

**Comment by vianasw** [comment]
I'm not really sure why but the issue is fixed by marking the version 3.0 [Preview] checkbox on the Direct Line config section. 

My old bot instance is still using version 1.1 and is working fine. In any case it was not clear at all that to fix it I had to use a preview version of the protocol.

**Comment by dandriscoll** [comment]
Hi @vianasw, in Direct Line 3.0, the From field has changed so it includes the site ID.

The correct way to handle this ID is to copy it from the Recipient field on inbound messages.

Direct Line happens to use your Bot Handle (from the Bot Framework registration portal) but the From and Recipient field always reflect the bot's account within the channel.
